samples/subsys/zbus/dyn_channel/src/consumer.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2022 Rodrigo Peixoto <rodrigopex@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "zbus_messages.h"
+
+#include <zephyr/logging/log.h>
+#include <zephyr/zbus/zbus.h>
+LOG_MODULE_DECLARE(zbus, CONFIG_ZBUS_LOG_LEVEL);
```

Comment:
Empty line after this? It's a bit nitpicky but I would suggest going over the various file and using a coherent style for `LOG_MODULE_DECLARE`. The one I like is having `LOG_MODULE_DECLARE` together with the "include" and then an empty line and the rest, so that the ZBUS ones stand out more... but just pick a style and stick to it.